libcontainer/rootfs_linux.go

```diff
@@ -1373,6 +1373,12 @@ func switchDockerDnsIP(config *configs.Config, pipe io.ReadWriter) error {
 		return err
 	}
 
+	//Fix: If there is not a default route
+	//Skip the DNS change because is not required
```

Comment:
Please adjust the comment to something like this: "Skip the DNS change if a default gateway isn't available."
And also point to this issue (issue-834) for future reference.